lib/phlex/rails/buffered.rb

```diff
@@ -4,37 +4,6 @@
 
 module Phlex::Rails
 	class Buffered < BasicObject
```

Comment:
Do we still need this class? It looks like only the `RadioButtonBuilder` is using it, everything else is using `Decorator`.

Reply:
Actually, we do still need this for some things such as our tag helper adapter.